### PR TITLE
Rustfmt `librustc_data_structures`

### PR DESCRIPTION
**Pull request by nikomatsakis**

### CODE REVIEW

File: src/jemalloc [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit e24a1a025a1f214e40eedafe3b9c7b1d69937922
+Subproject commit f84e30927284b0c500ed3eaf09e8e159da20ddaf
```
Comment (mitaa) [inline src/jemalloc]: This is probably unintentional.

Comment (nikomatsakis) [inline src/jemalloc]: Indeed :)

**Pull request merged**